### PR TITLE
Feat deauthorize logout api

### PR DESCRIPTION
**Pull request by mt-roberto**
- Adds `logout()` function to the SDK to logout from the current session.
- Adds tests for the new `logout()` function.
- Updates README file
- Refactored sample app to use Typescript

### CODE REVIEW

**Review by cmontgomerymt — APPROVED** [review]
LGTM

File: src/index.ts [hunk 1]
```diff
@@ -127,6 +127,20 @@ class LinkSDK {
     window.open(`https://${this.domains.myaccount}/${MY_ACCOUNT.PATHS.OAUTH}${params}`, newTab ? '_blank' : '_self');
   }
 
+  // Open My Account and logs you out from the current session
+  public logout({ newTab = false }: IMyAccountOptions = {}): void {
+    if (!this.isInitialized) {
+      throw new Error('SDK not initialized');
```
Comment (mt-neus) [inline src/index.ts]: This this track by sentry? if not (as is an sdk) what is the benefit of throwing an error?

**Review by mt-neus — COMMENTED** [review]
Do we need to track the dist folder?

File: src/index.ts (hunk 1 above)
Comment (mt-roberto) [reply src/index.ts]: Part of the SDK, it will warn the user that the SDK needs to be initialized before calling any other method.

**Review by mt-neus — APPROVED** [review]

**Review by mt-micky — APPROVED** [review]

**Pull request merged**